src/main/resources/reference.conf

```diff
@@ -1181,6 +1181,9 @@ resourceTypes = {
       view_journal = {
         description = "View dataset journal entries"
       }
+      view_snapshot_builder_config = {
```

Comment:
Will this have any effect if it's not part of any role? I would expect it to be added to the list of action for one of the roles below, like `admin`.

Reply:
Oh shoot, I thought I had committed that, mb.